src/NServiceBus.Core/Transports/Msmq/MsmqRoutingExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.Transports.Msmq
+{
+    using System.Collections.Generic;
+    using Routing;
+    using Settings;
+
+    /// <summary>
+    /// Provides MSMQ transport specific routing extensions.
+    /// </summary>
+    public static class MsmqRoutingExtensions
+    {
+        /// <summary>
+        /// Specify the machine name of a given logical endpoint.
```

Comment:
Can we add a stern warning here about using hardcoded values, or description of the scenarios where you'd want to do this?